python/gink/impl/lmdb_store.py

```diff
@@ -8,7 +8,7 @@
 from typing import Tuple, Iterable, Optional, Set, Union, Mapping, Callable
 from struct import pack
 from pathlib import Path
-from lmdb import open as ldmbopen, Transaction as Trxn, Cursor, BadValsizeError
+from lmdb import Environment, Transaction as Trxn, Cursor, BadValsizeError
```

Comment:
lmdbopen is just an alias for Environment, so this gives us typing for the lmdb methods :D

Reply:
cool